acmwebsite/controllers/schedule.py

```diff
@@ -0,0 +1,42 @@
+import datetime
+
+from icalendar import Calendar, Event
+from tg import expose, response
+
+from acmwebsite.lib.base import BaseController
+from acmwebsite.lib.helpers import log
+from acmwebsite.model import DBSession, Meeting
+
+
+class ScheduleController(BaseController):
+    def __init__(self):
+        self.meetings = DBSession.query(Meeting).order_by(Meeting.date).all()
+
+    @expose('acmwebsite.templates.schedule')
+    @expose('json', exclude_names=['page'])
+    def _index(self):
+        """Handle the schedule page."""
+
+        # Allow CORS
+        response.headers.add("Access-Control-Allow-Origin", "*")
```

Comment:
Do we need CORS for iCal to be embedded using JavaScript in other sites? If yes, then keep.

I presume that this will not be the only CORS thing we need... maybe we ought to add a decorator for this.

Reply:
I don't think we need this... It isn't even enabled for the actual `.ics` file. I just copy pasted it from the old controller. I'll remove it.